tests/client/crud_spec/framework.rs

```diff
@@ -298,7 +298,7 @@ macro_rules! check_coll {
         };
 
         let coll = match outcome_coll.name {
-            Some(ref str) => $db.collection(&str),
+            Some(ref s) => $db.collection(&s.clone()[..]),
```

Comment:
ah! must have been something weird tripping up the compiler in the intermediary state, while I was transitioning everything to serde_json. will fix.